src/test/scala/org/constellation/p2p/PeerAPITest.scala

```diff
@@ -71,19 +71,17 @@ class PeerAPITest
       val req = FinishedCheckpoint(CheckpointCache(None), Set.empty)
 
       Post("/finished/checkpoint", req) ~> addHeader("ReplyTo", reply) ~> peerAPI.postEndpoints ~> check {
-        status shouldEqual StatusCodes.OK
-        responseAs[FinishedCheckpointAck] shouldEqual FinishedCheckpointAck(true)
+        status shouldEqual StatusCodes.Accepted
```

Comment:
please add todo or something that we should check response as well, because in the test is written down, that this endpoint returns a particular response

Reply:
There is no response I have removed message phrase from test title